### ISSUE TITLE
Enable CONFIG_BT_HCIBTUSB_MTK in package linux5.x

### ISSUE THREAD

**Issue by rubin55** [void-linux/void-packages#34554]
### System

* xuname:  
  `Void 5.15.7_1 x86_64 AuthenticAMD notuptodate rF`
* package:  
  `linux5.15-5.15.7_1`

### Expected behavior
```
cat /boot/config-5.15.7_1 | grep CONFIG_BT_HCIBTUSB_MTK
CONFIG_BT_HCIBTUSB_MTK=m
```
### Actual behavior

```
cat /boot/config-5.15.7_1 | grep CONFIG_BT_HCIBTUSB_MTK
# CONFIG_BT_HCIBTUSB_MTK is not set
```

### Steps to reproduce the behavior

See above. This is simply a feature-request to enable `CONFIG_BT_HCIBTUSB_MTK` so that MediaTek based HCI bluetooth USB controllers start working (encountered this particular controller on my new AMD Ryzen 5850U based ThinkPad P14s).

**Comment by rubin55** [comment]
@leahneukirchen Is it possible to enable this also for the older kernels? Specifically I'm thinking about whichever version would have introduced the CONFIG_BT_HCIBTUSB_MTK flag, but at least 5.12, 5.13 and 5.14?

**Comment by leahneukirchen** [comment]
I wont bump EOL kernels, sorry.

**Comment by ericonr** [comment]
linux5.12 has already been removed from source repos, and linux5.13 and 5.14 will soon be removed.

**Comment by rubin55** [comment]
Ok fair enough. For posterity, I ran into an issue when both Bluetooth and Wi-Fi were enabled and thought it was a recent regression in the Linux kernel, hence the wish to have it available in older kernel versions.

Turns out this issue exists on older kernel versions equally. I wrote about the issue here: https://www.reddit.com/r/thinkpad/comments/rkvggg/thinkpad_p14s_gen2_amd_linux_515_mediatek_7961/

TL;DR; if you come here and think you have an issue with `mt7921e` and/or with `btusb`, check your wireless band with `iwconfig wlan0`. If its in the ~2.4Ghz range, you will find that it is very likely conflicting with your bluetooth stack which is also operating within the same range. Quick workaround: make sure you have a Wi-Fi router that can do ~5Ghz and connect your Wi-Fi on the ~5Ghz band, which avoids conflict with your bluetooth stack.

Other drivers (i.e., `iwl2200`) have a `bt_coex` option that seems to avoid this issue. `mt7921e` does not have this option (yet?). I emailed Sean Wang, who is one of the developers at MediaTek who works on `mt7921e` to share my experiences.
